test/test-keyman-api-service.ts

```diff
@@ -0,0 +1,29 @@
+import { responseBody } from "./fixtures/index";
```

Comment:
Unnecessary to include `index`:

```suggestion
import { responseBody } from "./fixtures";
```